uber/model_checks.py

```diff
@@ -813,28 +814,65 @@ def validate_phone(travel_plan):
 
 @prereg_validation.ArtShowApplication
 def max_panels(app):
-    if app.panels > c.MAX_ART_PANELS and app.panels != app.orig_value_of('panels'):
+    if app.panels > c.MAX_ART_PANELS and app.panels != app.orig_value_of('panels') or \
```

Comment:
Should this be checking against the sum of `app.panels + app.panels_ad` rather than each separately?

Reply:
Oh shoot, good catch!